recipe/meta.yaml

```diff
@@ -1,25 +1,24 @@
 {% set name = "vorbis" %}
-{% set version = "1.3.5" %}
-{% set sha256 = "6efbcecdd3e5dfbf090341b485da9d176eb250d893e3eb378c428a2db38301ce" %}
+{% set version = "1.3.6" %}
+{% set sha256 = "43fc4bc34f13da15b8acfa72fd594678e214d1cab35fc51d3a54969a725464eb" %}
 
 package:
   name: lib{{ name|lower }}
   version: {{ version }}
 
 source:
   fn:  lib{{ name }}-{{ version }}.tar.gz
-  url: http://downloads.xiph.org/releases/{{ name }}/lib{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xiph/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
The Github tar.gz contains the official CMakeList file, while the tar.gz from `downloads.xiph.org` does not contain it.

Reply:
It could be interesting to report this upstream - or is cmake only recommended to build on windows?

Reply:
I created that CMakeLists.txt file from scratch,  because at the time the libvorbis developer did not feel like maintaining another build system as he already had several in place. If there is an officially maintained CMakeLists.txt file now, please use that.

Reply:
> It could be interesting to report this upstream - or is cmake only recommended to build on windows?

What makes you say it would be only recommended on Windows? Did you see Windows-specific things in the CMakeList?

Reply:
Oh, I guess it's because of my issue title. Well yeah, I don't know why but cmake could find libvorbis correctly on Linux and OSX but not on Windows.

Anyway, I guess using the official CMakeList is safer now. It seems to work for me on Windows!

Reply:
> What makes you say it would be only recommended on Windows? Did you see Windows-specific things in the CMakeList?

Because some projects like e.g. libzmq support multiple build system but only really recommend cmake on windows.